Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -1228,330 +1221,277 @@ String CSSPrimitiveValue::stringValue() const
     }
 }
 
-NEVER_INLINE String CSSPrimitiveValue::formatInfiniteOrNanValue(ASCIILiteral suffix) const
+static NEVER_INLINE String formatNonfiniteValue(double number, ASCIILiteral suffix)
```

Comment:
Should this be `formatNonFiniteValue` with uppercase F?

Reply:
No. The word _nonfinite_ is a single English word that doesn’t even need a hyphen. I looked it up to be sure.

Reply:
OK, I thought it needed an hyphen.